### PR TITLE
Update Connect method example

### PR DESCRIPTION
**Pull request by dawei-wang**
Fix godotengine/godot-docs#9190

### CODE REVIEW

File: tutorials/best_practices/scene_organization.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ initialize it:
      .. code-tab:: csharp
 
        // Parent
-       GetNode("Child").Connect("SignalName", ObjectWithMethod, "MethodOnTheObject");
+       Callable.From(ObjectWithMethod.MethodOnTheObject)
```
Comment (dawei-wang) [inline tutorials/best_practices/scene_organization.rst:68]:
```suggestion
       Callable.From(ObjectWithMethod.MethodOnTheObject);
```

File: tutorials/best_practices/scene_organization.rst [hunk 2]
```diff
@@ -65,7 +65,7 @@ initialize it:
      .. code-tab:: csharp
 
        // Parent
-       GetNode("Child").Connect("SignalName", ObjectWithMethod, "MethodOnTheObject");
+       Callable.From(ObjectWithMethod.MethodOnTheObject);
```
Comment (AThousandShips) [inline tutorials/best_practices/scene_organization.rst:68]: This is not correct, you're not connecting to anything now, you're just creating a callable, it should be:
```suggestion
      GetNode("Child").Connect("SignalName", Callable.From(ObjectWithMethod.MethodOnTheObject));
```
I believe

File: tutorials/best_practices/scene_organization.rst [hunk 3]
```diff
@@ -65,7 +65,7 @@ initialize it:
      .. code-tab:: csharp
 
        // Parent
-       GetNode("Child").Connect("SignalName", ObjectWithMethod, "MethodOnTheObject");
+      GetNode("Child").Connect("SignalName", Callable.From(ObjectWithMethod.MethodOnTheObject));
```
Comment (AThousandShips) [inline tutorials/best_practices/scene_organization.rst:68]:
```suggestion
       GetNode("Child").Connect("SignalName", Callable.From(ObjectWithMethod.MethodOnTheObject));
```

**Comment by AThousandShips** [comment]
Thank you!

**Comment by mhilbrunner** [comment]
Cherrypicked to 4.2.

**Pull request merged**